packages/bpk-component-map/src/BpkMapMarker.js

```diff
@@ -100,6 +104,7 @@ BpkMapMarker.defaultProps = {
   large: false,
   onClick: null,
   type: MARKER_TYPES.primary,
+  buttonProps: {},
```

Comment:
Nitpick: This should be `null` by default as the Flow type indicates it's nullable. Having `{}` as the default means it can never be null.

Reply:
Updated :-)